app/src/main/java/com/rockthevote/grommet/data/db/model/Address.java

```diff
@@ -1,35 +0,0 @@
-package com.rockthevote.grommet.data.db.model;
-
-import android.os.Parcelable;
-import android.provider.BaseColumns;
-
-public abstract class Address implements Parcelable, BaseColumns {
-
-    public enum Type {
-        MAILING_ADDRESS("mailing_address"),
-        PREVIOUS_ADDRESS("previous_address"),
-        REGISTRATION_ADDRESS("registration_address"),
-        ASSISTANT_ADDRESS("assistant_address");
```

Comment:
Out of this, I can't find a previous use for:
- `previous_address` (Assuming it is `previous_registration_address` in the API?)
- `assistant_address` (Assuming it is `registration_helper.address` in the API?)